src/onchain-events/decode-event.spec.ts

```diff
@@ -0,0 +1,68 @@
+import { decodeEventData, decodeEventKey } from './decode-event';
+
+// To test and see the decoded event data structure, run:
+// yarn test --coverage src/onchain-events/decode-event.spec.ts
+
+test('decode deposit event data', () => {
+  const eventName = 'DepositEvent';
+  const eventData = '0x00ca9a3b00000000000000000000000000000000000000000000000000000001035354430353544300';
+  const result = decodeEventData(eventName, eventData);
+  console.log(result.toJS());
+});
```

Comment:
const result = decodeEventData(eventName, eventData);  
console.log(result.toJS());

thses two method should merge into one , and exported to the merged one